content/docs/csidriver/release/powerflex.md

```diff
@@ -12,6 +12,7 @@ description: Release notes for PowerFlex CSI driver
 
 ### Fixed Issues
 - [Fix the offline helm installation failure](https://github.com/dell/csm/issues/868)
+-  High/Critical Vulnerabilities in the csi-powerflex driver
```

Comment:
Please update the GitHub issue here.

Reply:
@khareRajshree  there is no github issue for it. It is not a fix in any functionality it will cause confusion.

Reply:
@khareRajshree : This section itself need to be removed